tools/chpl-language-server/test/basic_resolver.py

```diff
@@ -54,6 +54,68 @@ async def test_go_to_record_def(client: LanguageClient):
         await check_goto_type_def(client, doc, pos((3, 9)), pos((0, 7)))
 
 
+@pytest.mark.asyncio
+async def test_go_to_call_basic(client: LanguageClient):
+    """
+    Ensure that 'go to definition' on a type actually works.
```

Comment:
nit, comment is wrong

note that the comment should be updated for all the functions